config/locales/en.yml

```
@@ -1327,6 +1333,12 @@ en:
         public_search: 'Enable public search on Developer Portal'
         cms_escape_draft_html: 'Use Automatic Escaping for draft content'
         cms_escape_published_html: 'Use Automatic Escaping for published content'
+        change_plan_permission:
+          request: Request a plan change
+          direct: Change the plan directly
+          credit_card: Only request a plan change
+          request_credit_card: Request Credit Card entry for paid plans
+          none: Don't allow plan changes
```

After removing the hints (`"Change plan directly if developer has credit card, otherwise:"`), these options are not clear for me now. For instance, what's the difference between `"Request a plan change"` and `"Only request a plan change"`?

I think it needs some rewording. Just a suggestion:

```suggestion
          none: Don't allow plan changes
          request: Request a plan change
          direct: Change the plan directly
          credit_card: Change the plan if developer has CC, request a plan change otherwise
          request_credit_card: Change the plan if developer has CC, request CC details otherwise
```

---

Agreed, it's not clear at all :upside_down_face: 

I suggested this wording: https://github.com/3scale/porta/pull/3467#issuecomment-1641979386 (in Service Plans section). It is probably too long... but I think it's the only way to clarify the selected behavior.

---

Oh yes, true 👍.

---

I missed this 😬. Here is the new wording: https://github.com/3scale/porta/pull/3509